package/CHANGELOG

```diff
@@ -68,6 +79,8 @@ Fixes
   * fixed a bug in visualization.streamlines (Issue #1617)
   * Updated TPR parser for GROMACS 2018 (Issue #1796)
   * Fixed reading of PQR files generated by Gromacs (Issue #1804)
+  * core.groups.select_atoms now throws an error if the arbitrary atomgroups
```

Comment:
This can't go into the 18.0 changes, that's been frozen

Reply:
Okay. Didn't notice that. Changing it.